src/python/gudhi/representations/metrics.py

```diff
@@ -50,9 +50,9 @@ def _compute_persistence_diagram_projections(X, num_directions):
         num_directions (int): number of lines evenly sampled from [-pi/2,pi/2] in order to approximate and speed up the distance computation.
 
     Returns: 
-        list of n numpy arrays of shape (2*numx2): list of projected persistence diagrams.
+        list of n numpy arrays of shape (2*numxnum_directions): list of projected persistence diagrams.
```

Comment:
```suggestion
        list of n numpy arrays of shape (2*num,num_directions): list of projected persistence diagrams.
```
may be more readable (assuming that what you meant)?

Reply:
agreed